.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Practically.io All rights reserved
+
+name: Publish Docs
```

Comment:
I think this is good for now, do you think using vercel or GitHub pages will be good in the future.

Reply:
Yeah definitely